OtripleS.Portal.Web/Models/Courses/Exceptions/CourseServiceException.cs

```diff
@@ -0,0 +1,15 @@
+﻿// ---------------------------------------------------------------
+// Copyright (c) Coalition of the Good-Hearted Engineers
+// FREE TO USE AS LONG AS SOFTWARE FUNDS ARE DONATED TO THE POOR
+// ---------------------------------------------------------------
+
+using System;
+
+namespace OtripleS.Portal.Web.Models.Courses.Exceptions
+{
+    public class CourseServiceException : Exception
+    {
+        public CourseServiceException(Exception innerException)
```

Comment:
Change this to `Xeption`

Reply:
I still see Exception. The constructor is also Xeption. Also the same should be followed in all the wrapper exceptions.